rock_paper_scissors/api/models.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import (
+    Column,
+    Enum,
+    ForeignKey,
+    Integer,
+    String
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+ModelBase = declarative_base()
+
+
+class Game(ModelBase):
+    __tablename__ = 'game'
+
+    id = Column(Integer, primary_key=True)
+    player_1_id = Column(Integer, ForeignKey('user.id'))
+    player_2_id = Column(Integer, ForeignKey('user.id'))
+    rounds_limit = Column(Integer)
+    winner_id = Column(Integer, ForeignKey('user.id'), nullable=False)
+
+    player_1 = relationship('User', foreign_keys=[player_1_id])
+    player_2 = relationship('User', foreign_keys=[player_2_id])
+    winner = relationship('User', foreign_keys=[winner_id])
+
+    rounds_remaining = relationship('Round', foreign_keys='Round.id')
+    current_round = relationship('Round', foreign_keys='Round.id')
+
+
+class Round(ModelBase):
+    __tablename__ = 'round'
+
+    id = Column(Integer, primary_key=True)
+    game_id = Column(Integer, ForeignKey('game.id'))
+    winning_choice_id = Column(Integer, ForeignKey('choice.id'), nullable=False)
+
+    choices = relationship('Choice', foreign_keys=['Choice.round_id'])
+
+
+class User(ModelBase):
+    __tablename__ = 'user'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+
+    games_won = relationship('Game', foreign_keys=['Game.winner_id'])
+    rounds_won = relationship('Game', foreign_keys=['Round.winner_id'])
```

Comment:
You're correct on this. As discussed, we should instead make a `rounds_won` attribute (using `@property`). We can't base it on an iteration of associated `games_one`, however, because a player may have won rounds for a game which is currently incomplete.

Instead, we'll need to query for the data!

Specifically, we need all `Round`s where the `winning_choice` is associated with the user. When you get here, let me know and we can work on the query together.